scss/tokens/_component.scss

```diff
@@ -1,2 +1,94 @@
+$ouds-button-border-color-default-pressed-on-bg-emphasized: $ouds-color-action-primary-pressed-on-bg-emphasized !default;
+$ouds-button-content-color-default-disabled: $ouds-color-action-disabled !default;
+$ouds-button-content-color-default-disabled-on-bg-emphasized: $ouds-color-action-disabled-on-bg-emphasized !default;
+$ouds-button-content-color-default-enabled: $ouds-color-action-primary-enabled !default;
+$ouds-button-content-color-default-enabled-on-bg-emphasized: $ouds-color-action-primary-enabled-on-bg-emphasized !default;
+$ouds-button-content-color-default-focus: $ouds-color-action-primary-focus !default;
+$ouds-button-content-color-default-focus-on-bg-emphasized: $ouds-color-action-primary-focus-on-bg-emphasized !default;
+$ouds-button-content-color-default-hover: $ouds-color-action-primary-hover !default;
+$ouds-button-content-color-default-hover-on-bg-emphasized: $ouds-color-action-primary-hover-on-bg-emphasized !default;
+$ouds-button-content-color-default-loading: $ouds-color-action-primary-loading !default;
+$ouds-button-content-color-default-loading-on-bg-emphasized: $ouds-color-action-primary-loading-on-bg-emphasized !default;
+$ouds-button-content-color-default-pressed: $ouds-color-action-primary-pressed !default;
+$ouds-button-content-color-default-pressed-on-bg-emphasized: $ouds-color-action-primary-pressed-on-bg-emphasized !default;
+$ouds-button-content-color-minimal-disabled: $ouds-color-action-disabled !default;
+$ouds-button-content-color-minimal-disabled-on-bg-emphasized: $ouds-color-action-disabled-on-bg-emphasized !default;
+$ouds-button-content-color-minimal-enabled: $ouds-color-action-primary-enabled !default;
+$ouds-button-content-color-minimal-enabled-on-bg-emphasized: $ouds-color-action-primary-enabled-on-bg-emphasized !default;
+$ouds-button-content-color-minimal-focus: $ouds-color-action-primary-focus !default;
+$ouds-button-content-color-minimal-focus-on-bg-emphasized: $ouds-color-action-primary-focus-on-bg-emphasized !default;
+$ouds-button-content-color-minimal-hover: $ouds-color-action-primary-hover !default;
+$ouds-button-content-color-minimal-hover-on-bg-emphasized: $ouds-color-action-primary-hover-on-bg-emphasized !default;
+$ouds-button-content-color-minimal-loading: $ouds-color-action-primary-loading !default;
+$ouds-button-content-color-minimal-loading-on-bg-emphasized: $ouds-color-action-primary-loading-on-bg-emphasized !default;
+$ouds-button-content-color-minimal-pressed: $ouds-color-action-primary-pressed !default;
+$ouds-button-content-color-minimal-pressed-on-bg-emphasized: $ouds-color-action-primary-pressed-on-bg-emphasized !default;
+$ouds-button-size-icon: $ouds-size-icon-with-label-large-size-shorter !default;
+$ouds-button-size-icon-only: $ouds-size-icon-with-label-large-size-short !default;
+$ouds-button-size-loader: $ouds-size-icon-with-label-large-size-shorter !default;
+$ouds-button-space-column-gap-icon-end: $ouds-space-column-gap-with-arrow-short !default;
```

Comment:
On our side, it's more related to a dropdown, do we still consider it as a button variant ?